subsys/bluetooth/audio/bap_broadcast_source.c

```diff
@@ -799,10 +799,10 @@ int bt_bap_broadcast_source_reconfig(struct bt_bap_broadcast_source *source,
 			}
 		}
 
-		if (subgroup_stream_param_cnt < stream_cnt) {
+		if (subgroup_stream_cnt < subgroup_stream_param_cnt) {
```

Comment:
I can't comment on unedited code, but should the following comment on line 766 be changed to reflect this change? Seems like we are checking that the subgroup count is not exceeding the parameter count now.
`/* Verify that the parameter counts do not exceed existing number of subgroups and streams*/`

Reply:
> Seems like we are checking that the subgroup count is not exceeding the parameter count now.

No, now we are checking if `subgroup_stream_param_cnt` is not exceeding the `subgroup_stream_cnt` :) 

If we have 2 streams in the subgroup, then this returns an error if `subgroup_stream_param_cnt` is 3 or more.

Reply:
Yes you are right. Just making sure I understood it correctly :D But I was thinking backwards on the check. LGTM